collect_app/build.gradle

```diff
@@ -100,8 +100,8 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
             resValue("bool", "FIREBASE_CRASHLYTICS_ENABLED", "false")
         }
-        // Release build for the original ODK Collect app
-        odkCollectRelease {
+        // Release build for the official ODK Collect app
+        production {
```

Comment:
Why rename this? I think `odkCollectRelease` is as good as `production` and would tend to leave it as-is.

Reply:
Agreed that production is not a great name. I think `odkRelease` is better. I'd drop the `collect` because we already know it's a Collect release. What makes it a unique release is that it's an ODK release.